package.json

```diff
@@ -7,8 +7,14 @@
     "test": "yarn run format && yarn run lint-check && yarn run test:unit",
     "test:unit": "jest",
     "format": "prettier --write --list-different './src/'",
+    "precommit": "yarn lint-java && yarn clang-format",
     "lint-check": "eslint --ext '.js,.ts,.tsx' src/ && yarn prettier --check src/",
     "lint-check-cpp": "./scripts/cpplint.sh",
+    "lint-java": "node ./scripts/lint-java.js",
+    "clang-format": "yarn clang-format-ios && yarn clang-format-android && yarn clang-format-common",
+    "clang-format-ios": "find ios/ -iname *.h -o -iname *.m -o -iname *.cpp | xargs clang-format -i",
+    "clang-format-android": "find android/src/ -iname *.h -o -iname *.m -o -iname *.cpp | xargs clang-format -i",
+    "clang-format-common": "find Common/ -iname *.h -o -iname *.m -o -iname *.cpp | xargs clang-format -i",
```

Comment:
I think we should introduce `scripts/` directory at this point and just run the bash script instead of stacking package.json scripts.